pyproject.toml

```diff
@@ -1,9 +1,6 @@
-[build-system]
-requires = ["hatchling"]
-build-backend = "hatchling.build"
-
 [project]
 name = "adam_core"
+version = "0.1.0"
```

Comment:
Does this mean we remove the semantic git-hash based versioning? It might have already been removed with the hatch stuff. Just curious.

Reply:
I can potentially put it back. I was running into issues.